internal/datacoord/compaction_test.go

```diff
@@ -48,15 +39,19 @@ func TestCompactionPlanHandlerSuite(t *testing.T) {
 type CompactionPlanHandlerSuite struct {
 	suite.Suite
 
-	mockMeta  *MockCompactionMeta
-	mockAlloc *NMockAllocator
-	mockSch   *MockScheduler
+	mockMeta    *MockCompactionMeta
+	mockAlloc   *NMockAllocator
+	mockSch     *MockScheduler
+	mockCm      *MockChannelManager
+	mockSession *MockSessionManager
```

Comment:
NIT: mockSession naming is confusing since session could be service session. Maybe mockSessionMgr is better